hasjob/models/jobpost.py

```diff
@@ -546,13 +551,11 @@ def sort_score(self):
     def viewstats(self):
         now = datetime.utcnow()
         delta = now - self.datetime
-        if delta.days < 2:  # Less than two days
-            if delta.seconds < 21600:  # Less than 6 hours
-                return 'q', viewstats_by_id_qhour(self.id)
-            else:
-                return 'h', viewstats_by_id_hour(self.id)
+        hourly_stat_limit = 2 # days
+        if delta.days < hourly_stat_limit:  # Less than {limit} days
+            return 'h', viewstats_by_id_hour(self.id, hourly_stat_limit * 24)
```

Comment:
I moved the `limit` to the signature of `viewstats_by_id_hour` and `qhour` otherwise they were being define in 2 places. Kept the default limits.